aten/src/ATen/native/GatedLinearUnit.cpp

```diff
@@ -3,38 +3,38 @@
 #include <ATen/native/Activation.h>
 
 namespace at {
-namespace native {
-
-// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
-DEFINE_DISPATCH(glu_stub);
-// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
-DEFINE_DISPATCH(glu_backward_stub);
 
-Tensor& glu_out(const Tensor& self, int64_t dim, Tensor &result) {
+namespace meta {
+TORCH_META_FUNC(glu) (
```

Comment:
Any reason you decided to change `glu_out` to `glu`? I think here the implementation is meant for `glu_out` and `glu` is meant to leverage this method.

Reply:
That is the pattern here: https://github.com/pytorch/pytorch/pull/57619/files and indeed `TORCH_META_FUNC(glu_out)` does not correspond to a header declaration in `NativeMetaFunctions.h`. I am not sure if this is correct or if https://github.com/pytorch/pytorch/pull/57619/files is in error as well. @ezyang can probably advise.

Reply:
Defining the meta func for glu and not glu_out is correct!